### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by fadyanwar**
Fixing a couple of compile time errors on SPI usage sample code, a missing semicolon and a variable declaration

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/iot/pull/1870)

### CODE REVIEW

**Review by joperezr — APPROVED** [review]
Thanks for fixing!

**Comment by krwq** [comment]
I'm curious why is the variable nullable? `new` will never return non-null

File: src/devices/Mfrc522/README.md [hunk 1]
```diff
@@ -17,8 +17,8 @@ SpiConnectionSettings connection = new(0, 1);
 // Here you can use as well MfRc522.MaximumSpiClockFrequency which is 10_000_000
 // Anything lower will work as well
 connection.ClockFrequency = 5_000_000;
-SpiDevice spi = SpiDevice.Create(connection)
-mfrc522 = new(spi, 4);
+SpiDevice spi = SpiDevice.Create(connection);
+MfRc522? mfrc522 = new(spi, 4);
```
Comment (krwq) [inline src/devices/Mfrc522/README.md:21]: should this be `MfRc522 mfrc522 = new(spi, 4);` ?

Comment (fadyanwar) [reply src/devices/Mfrc522/README.md:21]: I was following the sample program as a guideline

**Pull request merged**